### ISSUE TITLE
Internal error: Property "data" does not exist in file x.php

### ISSUE THREAD

**Issue by simPod** [icanhazstring/phpstan-readonly-property#4]
I'm getting 
> Internal error: Property "data" does not exist in file x.php

error on similar code:

```php
class A {
    public function foo($val) {
        $x = new stdClass();
        $x->data = $val;
   }
}
```

basically when I'm assigning a value directly to any property. The extension seems not to distinguish between class own members and members of another instance.

**Comment by icanhazstring** [comment]
Hi, thanks. Will check that :)

**Comment by icanhazstring** [comment]
Fixed and tagged with 0.1.4. Thanks for reporting.

**Comment by simPod** [comment]
Using it and have no issues so far. Thx!
